docs/style.md

```diff
@@ -542,6 +542,25 @@ in Flow, or a design limitation of Flow, or an issue in the Flow types
 provided by Immutable.js, or some combination.)
 
 
+<div id="immutable-no-object-as-map" />
+
+**Don't construct an `Immutable.Map` with an object-as-map**: Whenever you
```

Comment:
Should get an entry in the table of contents, following the pattern of the others.

(In particular, that provides the only convenient way to get these nice permalinks like `…#immutable-no-object-as-map` when viewing the document on GitHub.)